test/blackbox-tests/test-cases/melange/enabled_if.t

```diff
@@ -0,0 +1,55 @@
+`enabled_if` in `melange.emit`
+
+  $ cat > dune-project <<EOF
+  > (lang dune 3.8)
+  > (using melange 0.1)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target out)
+  >  (emit_stdlib false)
+  >  (enabled_if %{bin-available:melc}))
+  > EOF
+  $ cat > x.ml <<EOF
+  > let () = Js.log "hello"
+  > EOF
+
+Not available until dune lang 3.9
```

Comment:
If this field needs versioning, it should be using the melange syntax rather than the dune syntax.